dashboard/config/levels/custom/dance/hoai2025-dance-instructions.level

```diff
@@ -0,0 +1,103 @@
+<Dancelab>
+  <config><![CDATA[{
+  "published": true,
+  "game_id": 63,
+  "created_at": "2025-10-19T04:43:34.000Z",
+  "level_num": "custom",
+  "user_id": 1,
+  "properties": {
+    "skin": "dance",
+    "block_pools": [
+      "Dancelab",
+      "GeneratedDancers"
+    ],
+    "helper_libraries": [
+      "DanceLab"
+    ],
+    "hide_animation_mode": true,
+    "show_type_hints": true,
+    "encrypted": "false",
+    "embed": "false",
+    "instructions_important": "false",
+    "submittable": "false",
+    "never_autoplay_video": "false",
+    "disable_param_editing": "true",
+    "use_modal_function_editor": "false",
+    "hide_share_and_remix": "true",
+    "disable_if_else_editing": "false",
+    "free_play": "false",
+    "validation_enabled": "false",
+    "show_debug_watch": "false",
+    "expand_debugger": "false",
+    "debugger_disabled": "false",
+    "hide_pause_button": "false",
+    "default_song": "introtoshamstep_47SOUL",
+    "long_instructions": "##Welcome to the party!\r\n- Connect a dancer block to When Run\r\n- Click Run to start dancing.",
+    "uses_lab2": "true",
+    "guide_mode": "instructions",
+    "toolbox_definition": {
+      "contents": [
+        {
+          "kind": "block",
+          "type": "GeneratedDancers_makeAnonymousDanceSprite",
+          "id": "vzvF:!o_2n#y..Ztz|Vu",
+          "fields": {
+            "COSTUME": "\"GENERATED_DANCER\"",
+            "LOCATION": "<field name=\"LOCATION\">{x: 200, y: 100}</field>"
+          },
+          "enabled": true
+        }
+      ],
+      "kind": "flyoutToolbox"
+    },
+    "start_sources": {
+      "source": {
+        "blocks": {
+          "blocks": [
+            {
+              "id": "when-run-block",
+              "type": "Dancelab_whenRun",
+              "deletable": false,
+              "movable": false
+            }
+          ]
+        }
+      },
+      "selectedSong": "asitwas_harrystyles"
+    },
+    "validation_code": "if (nativeAPI.getTime(\"measures\") > 5) {\r\n  if (sprites.length === 0) {\r\n    nativeAPI.fail(\"dance-fail\");\r\n  } else {\r\n    nativeAPI.pass();\r\n  }\r\n}",
```

Comment:
Looks like our new validation conditions are still interacting with the old validation code system, right? Worth deprecating the old one at some point?